packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.generated.ts

```diff
@@ -64,26 +64,26 @@ use_old_VariableDeclaration_AllowTombstoneRequestHeaderKey(
 /*
 * Validate forward compat by using old type in place of current type
 * If breaking change required, add in package.json under typeValidation.broken:
-* "TypeAliasDeclaration_CompatModeBehavior": {"forwardCompat": false}
+* "RemovedTypeAliasDeclaration_CompatModeBehavior": {"forwardCompat": false}
 */
 declare function get_old_TypeAliasDeclaration_CompatModeBehavior():
     TypeOnly<old.CompatModeBehavior>;
-declare function use_current_TypeAliasDeclaration_CompatModeBehavior(
+declare function use_current_RemovedTypeAliasDeclaration_CompatModeBehavior(
```

Comment:
Yeah since I'm 99.99% sure no one is using this it seems fine to just remove.  Typically we should deprecate though.